docs/contributing.rst

```diff
@@ -70,3 +75,28 @@ To compile documentation into webpages, run
 under the ``docs/`` directory. The generated webpages are in ``docs/_build`` and can be viewed with browsers.
 
 Chinese documentation is in https://tianshou.readthedocs.io/zh/latest/.
+
+
+Documentation Test
```

Comment:
You mean "documentation generation" ? I think it would be more clear.

Reply:
Doc generation is `cd docs/; make html;`, the doc8, pydocstyle, and `make -W` test the potential error when generating documentation.

Reply:
Ok good to know such features exist !